sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -500,6 +502,17 @@ public OutT apply(KafkaRecord<KeyT, ValueT> record) {
             return new KafkaConsumer<>(config);
           }
         };
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("topics", Joiner.on(",").join(getTopics())));
```

Comment:
Yeah I missed the partitions or topics part, will do.
Are we sure we want to change the name of the key depending on the number of topics? Shouldn't display data keys be consistent?